src/utils/command.rs

```diff
@@ -43,6 +43,7 @@ pub fn run_command(command: Command) -> Result<ActionResult, ActionError> {
     match std::process::Command::new(&command.name)
         .envs(&command.env)
         .args(&command.args)
+        .current_dir(&command.dir)
```

Comment:
I am a Rust newbie, what would be a good way to set this current_dir only if dir is set?

Something like setting it to current dir if it is not set and setting it to command.dir if it is set?

Reply:
You could make the `dir` field on the struct `Option<String>` and then use `.current_dir(&command.dir.clone().unwrap_or(std::env::current_dir())`

Reply:
The more idiomatic way would be to `impl Default` for the `Command` struct and provide defaults. It's a little more work.

https://doc.rust-lang.org/std/default/trait.Default.html

Reply:
I'm happy with either approach, things can be cleaned in due course. Lets get you this feature shipped 😄